### ISSUE TITLE
Listening to ViewState is not as clear now when using view.set() and view.get();

### ISSUE THREAD

**Issue by mandragorn** [vecnatechnologies/backbone-torso#170]
Its not entirely clear that these 2 methods are dealing with the same object.

this.set('someProperty', 'someValue');
this.listenTo(this.viewState, 'change', this.render);

**Comment by mandragorn** [comment]
Ideas:
this.listenToViewState('change', this.render);

this.on('change', this.render);

extend({
  renderOnViewStateChange: false
});

extend({
  viewEvents: {
    'change', 'render'
  }
});

**Comment by mandragorn** [comment]
Leaning towards the last option, but name? - viewEvents or viewStateEvents?

**Comment by kentmw** [comment]
I think that "change" should be the indicator for whether the view.on() call is routed to itself or to the view state. View's events don't have an automated "change" event so it won't conflict there, and 99% use case for listening to the view state is change in it's state. The only thing not captured is events like "reset" or the view state and in those cases I think it's fine having this.listenTo(this.viewState...)

basically, we should wrap the view's on call, look to see if it is `change` or starts with `change:` and route it to the view state otherwise route it like normal

**Comment by kentmw** [comment]
Actually, looking at the [events](http://backbonejs.org/#Events-catalog), a Torso Cell shouldn't emit any events other besides "change" making this even better. A cell doesn't emit persistence-related events, nor collection-related events. Perhaps the "destroy" event would be used, but using the view's _dispose is probably better anyway - regardless really small use case

**Comment by mandragorn** [comment]
that makes sense to me.  Make sure the documentation is clear that 'change' is a reserved event on a torso view.
